runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5132,6 +5132,56 @@ class INTERPRETER_CLASS
 	}
 #endif /* JAVA_SPEC_VERSION >= 16 */
 
+#if JAVA_SPEC_VERSION >= 19
+	/* jdk.internal.vm.Continuation: private native boolean enterImpl(); */
+	VMINLINE VM_BytecodeAction
+	enterContinuationImpl(REGISTER_ARGS_LIST)
+	{
+		VM_BytecodeAction rc = EXECUTE_BYTECODE;
+		jboolean res = JNI_FALSE;
+
+		j9object_t continuationObject = *(j9object_t*)_sp;
+
+		buildInternalNativeStackFrame(REGISTER_ARGS);
+		updateVMStruct(REGISTER_ARGS);
+
+		res = enterContinuation(_currentThread, continuationObject);
+
+		VMStructHasBeenUpdated(REGISTER_ARGS);
+		if (VM_VMHelpers::exceptionPending(_currentThread)) {
+			rc = GOTO_THROW_CURRENT_EXCEPTION;
+			goto done;
+		}
+
+		restoreInternalNativeStackFrame(REGISTER_ARGS);
+		returnSingleFromINL(REGISTER_ARGS, res, 1);
+done:
+		return rc;
+	}
+
+	/* jdk.internal.vm.Continuation: private static native boolean yieldImpl(); */
+	VMINLINE VM_BytecodeAction
+	yieldContinuationImpl(REGISTER_ARGS_LIST)
+	{
+		VM_BytecodeAction rc = EXECUTE_BYTECODE;
+		buildInternalNativeStackFrame(REGISTER_ARGS);
+		updateVMStruct(REGISTER_ARGS);
+
+		/* store the current Continuation state and swap to carrier thread stack */
+		yieldContinuation(_currentThread);
+
+		VMStructHasBeenUpdated(REGISTER_ARGS);
+		restoreInternalNativeStackFrame(REGISTER_ARGS);
+
+		/* its going to return as if it were returning from continuation.enter()
+		 * so we need to push the boolean return val
+		 */
+		returnSingleFromINL(REGISTER_ARGS, JNI_FALSE, 1);
```

Comment:
Should `slotCount` be `0` since the method is static?

Reply:
The slotCount 1 is to simulate a return to carrier thread from call of `enterImpl` which is non-static